BeagleBone_black/src/navigation/navigation.cpp

```diff
@@ -135,12 +135,13 @@ void Navigation::update(ImuArray imus, ProximityArray proxis, DataPoint<uint32_t
 
 void Navigation::calibrationUpdate(ImuArray imus)
 {
+  // Online mean algorithm
+  ++num_gyro_samples_;
   for (unsigned int i = 0; i < data::Sensors::kNumImus; ++i) {
-    g_ += imus[i].acc.value;
-    gyro_offsets_[i] += imus[i].gyr.value;
+    ++num_gravity_samples_;
```

Comment:
Why is `num_gravity_samples_` incremented in the loop body but `num_gyro_samples_` out of the loop?